testsuite/cluster-test/src/experiments/performance_benchmark.rs

```diff
@@ -38,6 +38,12 @@ pub struct PerformanceBenchmarkParams {
     help = "Duration of an experiment in seconds"
     )]
     pub duration: u64,
+    #[structopt(
+        long,
+        default_value = "0",
+        help = "Set fixed tps during perf experiment"
+    )]
+    pub tps: u64,
```

Comment:
Ideally you want to avoid magic values. Right now this magic value is 0, meaning that 'don't use this field.
Instead, you can have this field as `Option<u64>`